evalml/tests/data_checks_tests/test_datetime_format_data_check.py

```diff
@@ -107,7 +116,7 @@ def test_datetime_format_data_check_typeerror_uneven_intervals(
             dates = pd.Series(y.index)
         else:
             dates = X[datetime_column]
-        ww_payload = infer_frequency(dates, debug=True, window_length=5, threshold=0.8)
+        ww_payload = infer_frequency(dates, debug=True, window_length=4, threshold=0.4)
```

Comment:
It could be useful to add `WINDOW_LENGTH` and `THRESHOLD` variables to the beginning of the test file with these values set so if we change this in the future, we only have to change it in one place